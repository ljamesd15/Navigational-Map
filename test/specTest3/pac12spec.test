# Testing the specifications using pac12_v2.tsv file

# Load the graph.
LoadGraph pac12 pac12_v2.tsv

# Find a normal path.
FindPath pac12 Huskies Cougars

# Find a path with more edges but smaller cost.
FindPath pac12 Huskies Ducks

# Find a path with equal cost but lexicographically shorter.
FindPath pac12 Huskies Trojans

# Find a path where start and target node are the same
FindPath pac12 Huskies Huskies

# Attempt to find path which does not exist.
FindPath pac12 Huskies Utes

# Find path where one character doesn't exist.
FindPath pac12 Huskies Brown_University

# Find path where both characters don't exist.
FindPath pac12 Western_Washington Brown_University

# Find a path on a graph that does not exist.
FindPath graphDoesNotExist Huskies Huskies