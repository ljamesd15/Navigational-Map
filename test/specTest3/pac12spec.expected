# Testing the specifications using pac12_v2.tsv file

# Load the graph.
loaded graph pac12

# Find a normal path.
path from Huskies to Cougars:
Huskies to Cougars with weight 0.333
total cost: 0.333

# Find a path with more edges but smaller cost.
path from Huskies to Ducks:
Huskies to Cougars with weight 0.333
Cougars to Ducks with weight 0.500
total cost: 0.833

# Find a path with equal cost but lexicographically shorter.
path from Huskies to Trojans:
Huskies to Bears with weight 1.000
Bears to Bruins with weight 0.333
Bruins to Trojans with weight 0.333
total cost: 1.667

# Find a path where start and target node are the same
path from Huskies to Huskies:
total cost: 0.000

# Attempt to find path which does not exist.
path from Huskies to Utes:
no path found

# Find path where one character doesn't exist.
unknown character Brown University

# Find path where both characters don't exist.
unknown character Western Washington
unknown character Brown University

# Find a path on a graph that does not exist.
The graph 'graphDoesNotExist' does not exist.